tonic/functional/__init__.py

```diff
@@ -2,45 +2,33 @@
 from .denoise import denoise_numpy
 from .drop_event import drop_event_numpy
 from .drop_pixel import drop_pixel_numpy, identify_hot_pixel
-from .flip_lr import flip_lr_numpy
```

Comment:
Why did you get rid of these ?

Reply:
because the functional transform was just 3 lines of code, I just merged it into the transforms.py directly